### PR TITLE
cmd/atlascmd: env support for migrate commands

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

**Review by a8m — COMMENTED** [review]
The new change adds a lot of complexity which is a bit surprising. Don't you prefer to use cobra hooks (e.g. pre-run) to set the flags?

File: cmd/atlascmd/schema.go [hunk 1]
```diff
@@ -31,15 +39,22 @@ var (
 		Long:  "The `atlas schema` command groups subcommands for working with Atlas schemas.",
 	}
 
+	// SchemaFlags are common flags used by schema commands.
+	SchemaFlags struct {
+		URL     string
+		Schemas []string
+
+		// Deprecated: DSN is an alias for URL.
+		DSN string
```
Comment (a8m) [inline cmd/atlascmd/schema.go:48]: Why it's here? We should not accept it in the project file.

Comment (rotemtam) [reply cmd/atlascmd/schema.go:48]: I moved flags common between inspect and apply here. It's not supported in the project file

**Review by a8m — COMMENTED** [review]
Looks good, minor questions before I approve this. 

Thanks

**Review by a8m — APPROVED** [review]

**Pull request merged**